src/Integration/Connection/ConnectionController.cs

```diff
@@ -41,7 +41,7 @@ internal class ConnectionController : HostedCommandControllerBase, IConnectionIn
     {
         private readonly IHost host;
         private readonly IConnectionInformationProvider connectionProvider;
-        private readonly IIntegrationSettings settings;
```

Comment:
Is the _IIntegrationSettings_ interfaces still used anywhere? Looks like it might only be used in tests.

Reply:
It was deleted a couple of months ago when there was a refactoring from scanner team but all references were not fixed. Causing a null reference exception.